catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -248,7 +249,22 @@ public void setProcessingMechanism(String processingMechanism) {
      * @param attributeOverrides - a list of attributes to override
      */
     public void setAttributeOverrides(List<String> attributeOverrides) {
-        this.attributeOverrides = attributeOverrides;
+        Map<String, Serializable> attributeOverrideMap = new HashMap<>();
+        for (String s : attributeOverrides) {
+            String[] keyValue = s.split("=");
```

Comment:
If we expect there to only be two, why not just add the limit to the split (`s.split("=",  2)`) such that we ignore the rest of equals and include them in the value.